core/targets.py

```
@@ -708,7 +531,7 @@ def get_pregnancy_targets(**profile) -> dict:
     }
 
 
-def calculate_pre_post_workout(workout_type: str, duration: int) -> dict:  # noqa: ARG001
+def calculate_pre_post_workout(_workout_type: str, duration: int) -> dict:  # noqa: ARG001
```

Ruff (0.13.3)

Unused `noqa` directive (unused: `ARG001`)

Remove unused `noqa` directive

(RUF100)